### PR TITLE
docs(README): Make the Helm installation command simpler.

### PR DESCRIPTION
**Pull request by pangruoran**
**What this PR does / why we need it**:

The `ingressController.installCRDs` has been removed since [1].
For the getting started documentation, it would be more helpful to provide a simpler command.

1: https://github.com/Kong/charts/commit/bf98a5cbae44af1f30c72f0d0d4f1910a38a0d8c

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,10 +45,7 @@ Setting up Kong for Kubernetes is as simple as:
 
 ```shell
 # Using Helm
-helm repo add kong https://charts.konghq.com
-helm repo update
-
-helm install kong/kong --generate-name --set ingressController.installCRDs=false
+helm install kong --generate-name -n kong --create-namespace --repo https://charts.konghq.com
```
Comment (pmalek) [inline README.md:48]: Will this `update` the repo cache 🤔 ?

If not then it's better to use the separate steps as before to ensure users get the latest available version

Comment (pangruoran) [reply README.md:48]: Yes.

If we pass the repository path via the `--repo` flag, Helm will save it to a temporary directory and fetch the latest index (similar to `helm repo update`), ensuring that it always has access to the latest version.

Comment (pmalek) [reply README.md:48]: Alright, good 👍 

While we're at it, I have thought: why don't we advertize `ingress` chart here already? @mheap Are we ready to do it?

`kong` chart is still installing the sidecar deployment which we're trying to deprecated. With `ingress` we'll promote the split deployments installation

How about:

```suggestion
helm install kong --namespace kong --create-namespace --repo https://charts.konghq.com ingress
```

Comment (pmalek) [reply README.md:48]: Given that https://github.com/Kong/kubernetes-ingress-controller/issues/4671 says that 

> Make `kong/ingress` the only recommended way of installing KIC in the documentation.

I believe this is the change we'd want to make

Comment (pangruoran) [reply README.md:48]: Thank you for pointing this out, I will try deploying it using the ingress helm chart

**Review by pmalek — APPROVED** [review]

**Pull request merged**